components/playground/instance/tidb.go

```diff
@@ -34,9 +34,13 @@ type TiDBInstance struct {
 }
 
 // NewTiDBInstance return a TiDBInstance
-func NewTiDBInstance(binPath string, dir, host, configPath string, id, port int, pds []*PDInstance, enableBinlog bool, isDisaggMode bool) *TiDBInstance {
+func NewTiDBInstance(binPath string, dir, host, configPath string, id, port int, pds []*PDInstance, enableBinlog bool, isDisaggMode bool, tiproxys []*TiProxy) *TiDBInstance {
 	if port <= 0 {
-		port = 4000
+		if len(tiproxys) > 0 {
```

Comment:
it may be confusing if use different default port

Reply:
TiProxy will be used as the **default port**, which is 4000. Thus tidb must be started at a different port, e.g. 6000.

Reply:
hi, sorry for jump in, may I know why tiproxy use 4000 as default port? 
we all know 3306 is mysql default port, 1521 is oracle default port, 5432 is postgresql default port, and 6033 is proxysql default port.
if tiproxy, a new middleware/proxy for tidb, use 6000 as default port, that could be much better.

Reply:
Tiproxy use 6000 as the default port. But we want to enable tiproxy by default. So it used the default port of tidb - 4000.

Reply:
It is confusing if using default port in different case.I think remain 6000 as tiproxy default port and modify welcome message to 6000 is enough